### PR TITLE
add nft shape basic test, change title of scenes (now is only description without coordinates)

### PR DESCRIPTION
**Pull request by sdilauro**
Added a TODO to issues to be careful about this test.

### CODE REVIEW

File: 54,-52-material/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "start": "sdk-commands start"
   },
   "devDependencies": {
-    "@dcl/js-runtime": "https://sdk-team-cdn.decentraland.org/@dcl/js-sdk-toolchain/branch/psquad/test-framework-tool/@dcl/js-runtime/dcl-js-runtime-7.3.36-7291127796.commit-b299b0d.tgz",
+    "@dcl/js-runtime": "7.3.36-7291127796.commit-b299b0d",
```
Comment (leanmendoza) [inline 54,-52-material/package.json:11]: this shouldn't be commited

File: 54,-52-material/src/tests/material/index.test.ts [hunk 2]
```diff
@@ -444,3 +448,39 @@ test('material: rock wall texture with bump texture: if exist a reference snapsh
     Vector3.create(8, 1, 8)
   )
 })
+
+test('material: avatar portrait', async function (context) {
```
Comment (leanmendoza) [inline 54,-52-material/src/tests/material/index.test.ts:452]: skipTest while it's not working
add todo comment

File: 54,-68-nft-shape/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "nft-shape",
+  "version": "protocol-squad",
+  "description": "SDK7 Template scene",
+  "scripts": {
+    "build": "sdk-commands build --skip-install",
+    "deploy": "sdk-commands deploy",
+    "start": "sdk-commands start"
+  },
+  "devDependencies": {
+    "@dcl/js-runtime": "7.3.36-7291127796.commit-b299b0d",
```
Comment (leanmendoza) [inline 54,-68-nft-shape/package.json:11]: commit the same `package.json` placed in the other tests

File: testing-library/src/testing/index.ts [hunk 4]
```diff
@@ -13,6 +13,7 @@ declare let require: any
 export const test: TestDefinitionFunction = DEBUG
   ? /* @__PURE__ */ createTestFunction()
   : /* @__PURE__ */ () => {}
+export const skipTest: TestDefinitionFunction = () => {}
```
Comment (leanmendoza) [inline testing-library/src/testing/index.ts:16]: is this duplicated? I remember this had been added

**Review by leanmendoza — CHANGES_REQUESTED** [review]

File: 54,-68-nft-shape/package.json (hunk 3 above)
Comment (sdilauro) [reply 54,-68-nft-shape/package.json:11]: fixed

File: 54,-52-material/package.json (hunk 1 above)
Comment (sdilauro) [reply 54,-52-material/package.json:11]: fixed

File: testing-library/src/testing/index.ts (hunk 4 above)
Comment (sdilauro) [reply testing-library/src/testing/index.ts]: fixed

**Pull request merged**